docs/development/feature.md

```diff
@@ -568,7 +578,8 @@ const register = (): WorkflowConfiguration => {
     displayName: "Amiibo",
     routes: {
       landing: {
-        path: "/",
+  // highlight-next-line
```

Comment:
https://clutch.sh/docs/development/feature/#test-1 says to navigate to `http://localhost:3000/amiibo/lookup`